msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1390,6 +1403,7 @@ public void run() {
                             callback
                     );
 
+                    silentTokenCommand.setPublicApiId(PublicApiId.LOCAL_ACQUIRE_TOKEN_SILENT);
```

Comment:
As discussed, 
if you want to track what API is calling, you could rename this function to internal and take an API constant as another parameter, then pass that parameter to setPublicApiId. Should be a pretty straighforward and simple change.

You could do it in this PR or a separate one. That one i'll let you make a call.

Reply:
Yeah, sounds good. I think I'll do that in a separate PR after this gets merged in.